src/System.IO.UnmanagedMemoryStream/tests/UmsSecurityTest.cs

```diff
@@ -27,6 +27,8 @@ public static void ChangePositionViaPointer()
                         stream.PositionPointer = stream.PositionPointer - 1;
                     });
 
+                    Assert.Throws<ArgumentOutOfRangeException>(() => stream.PositionPointer = (byte*)ulong.MaxValue);
```

Comment:
This is fragile. It the OS allocates `data` at very high address (e.g. `ulong.MaxValue - 0x10000000`), this test will fail. You may want to move this check into the OverflowPositionPointer test where the stream is initialized with a predictable base address.